transport/http/client.go

```diff
@@ -59,6 +61,13 @@ func SetClientBefore(before ...RequestFunc) ClientOption {
 	return func(c *Client) { c.before = before }
 }
 
+// SetClientAfter sets the ClientResponseFuncs applied to the incoming HTTP
+// request prior to it being decoded. This is useful for obtaining anything off
+// of the response and adding onto the context prior to decoding.
+func SetClientAfter(after ...ClientResponseFunc) ClientOption {
```

Comment:
Sorry for the nit :) just noticing this now. In transport/http.Server, none of the functional options begin with **Set**, but in Client, they all do. I guess this is an unintended consequence of the first functional option on the client being SetClient (to set the HTTP client). Can I ask you to go above-and-beyond one last time and make the following renames in http/transport.Client?

- SetClient → SetClient (no change)
- SetClientBefore → ClientBefore
- SetClientAfter → ClientAfter
- SetBufferedStream → BufferedStream